setup.py

```diff
@@ -14,7 +14,7 @@
 
 CYTHON_MACROS = [('CYTHON_TRACE', '1')] if CYTHON_DEBUG else None
 
-VERSION = '0.3.2'
+VERSION = '0.4.1'
```

Comment:
Did we have a 0.4.0 already?

Reply:
v0.4.0 was a release that was made, however the `VERSION` remained unchanged from v0.3.2. That docker was built, however it wasn't pushed to PyPI

Reply:
Nope that's fine